### PR TITLE
Hw18

### PR DESCRIPTION
**Pull request by un-install**

### CODE REVIEW

File: src/main/resources/hw16-screenshots/hw17.sql [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+inner join salesreps s on o.rep = s.empl_num;
+    
+select amount, amount, amount/credit_limit, amount/quota from orders o
+inner join customers c on o.cust = c.cust_num
+inner join salesreps s on o.rep = s.empl_num;
+
+select distinct c.* from customers c
+inner join orders o on c.cust_num = o.cust
+inner join products p on o.product = p.product_id
+and p.description like '%A%' or p.description like '%a%' or p.description like '%�%' or p.description like '%�%';
```
Comment (MateAcademy) [inline src/main/resources/hw16-screenshots/hw17.sql]: Why in the homework you add step 32-26, this is the previous homework?

Comment (MateAcademy) [reply src/main/resources/hw16-screenshots/hw17.sql]: Wery good homewok!

Comment (un-install) [reply src/main/resources/hw16-screenshots/hw17.sql]: Oh, didnt notice. Sims like I pushed in existed branch. Sorry

**Comment by un-install** [comment]
Watch from 3rd commit

File: src/main/resources/hw18.sql [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+--8 with subquery
+select max(orders_count), min(orders_count) from
+(select count(*) as orders_count from orders
+where rep in (select manager from salesreps group by manager having count(*) > 2)
+group by rep);
+
+--9
+select * from products
+where regexp_like(description,'[аАaA].*[аАaA]');
```
Comment (SergeyPasko) [inline src/main/resources/hw18.sql]: Very good

**Pull request closed without merging**